pyomo/solvers/tests/checks/test_BARON.py

```diff
@@ -0,0 +1,32 @@
+"""Tests the BARON interface."""
+import pyutilib.th as unittest
+from pyomo.environ import (ConcreteModel, Constraint, Objective, Var, log10,
+                           minimize)
+from pyomo.opt import SolverFactory, TerminationCondition
+
+# I don't know how you figure out if BARON is available
+baron_available = True

```

Comment:
@ghackebeil: for this test, it should be well within what the unlicensed version of BARON allows - so do we really need to check the license?